provision/service.go

```diff
@@ -382,7 +384,7 @@ func (ps *provisionService) recover(e *error, ths *[]SDK.Thing, chs *[]SDK.Chann
 		}
 	}
 
-	if errors.Contains(err, SDK.ErrFailedWhitelist) || errors.Contains(err, ErrGatewayUpdate) {
+	if errors.Contains(err, ErrFailedWhitelist) || errors.Contains(err, ErrGatewayUpdate) {
```

Comment:
Let's stick to `Things` terminology: `ErrThingUpdate`.

Reply:
I updated the error